userInformation_program.c

```diff
@@ -0,0 +1,208 @@
+#include<stdio.h>
+#include<stdbool.h>
+#include<stdlib.h>
+#include<string.h>
+
+struct MyStructure
+{
+    int ID;
+    char name[100];
+    int age;
+};
+
+void create(FILE *file){
+    struct MyStructure s;
```

Comment:
Focus on variable and function naming so that it can be understandable.
Here, there can be a better name of a structure than "s".